### PR TITLE
11180_seatTypeManage

### PR DESCRIPTION
**Pull request by ahxtah1996**
https://edu-redmine.sun-asterisk.vn/issues/11180

### CODE REVIEW

**Comment by ahxtah1996** [comment]
ready

File: app/Http/Controllers/SeatTypeController.php [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    public function index(Request $request)
+    {
+        if ($request->ajax()) {
+            $data = Seat_type::latest()->get();
+
+            return Datatables::of($data)
+                    ->addIndexColumn()
+                    ->addColumn('action', function($row){
+                            $btn = '<a href="javascript:void(0)" data-toggle="tooltip"  data-id="' . $row->id . '" data-original-title="Edit" class="edit btn btn-primary btn-sm editSeatType">Edit</a>';
+                            $btn = $btn . ' <a href="javascript:void(0)" data-toggle="tooltip"  data-id="' . $row->id . '" data-original-title="Delete" class="btn btn-danger btn-sm deleteSeatType">Delete</a>';
```
Comment (buihanh-fr) [inline app/Http/Controllers/SeatTypeController.php]: i18n

File: app/Http/Controllers/SeatTypeController.php [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+     * Store a newly created resource in storage.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return \Illuminate\Http\Response
+     */
+    public function store(Request $request)
+    {
+        Seat_type::updateOrCreate(['id' => $request->seat_type_id],
+        ['name' => $request->name,
+        'note' => $request->note]);
```
Comment (buihanh-fr) [inline app/Http/Controllers/SeatTypeController.php]: convention

**Comment by ahxtah1996** [comment]
ready

**Pull request merged**